### PR TITLE
Remove "pause" dialog and extra filters.

### PR DESCRIPTION
**Pull request by nathan130200**
- Added an contrast and brightness filters to enhance
- Fix object fit to `fill` (eg. non 16:9 resolutions or even running xcloud without fullscreen mode)
- Remove pause dialog ("Click here to return to game")

### CODE REVIEW

**Comment by redphx** [comment]
Thank you for the PR, but I think I can only accept the "Remove pause dialog" for now. Other changes will be added later when I add a Settings dialog.

**Comment by nathan130200** [comment]
No problem =D

File: better-xcloud.user.js [hunk 1]
```diff
@@ -21,7 +21,10 @@ div[class*=Grip-module__container] {
 
 /* Add video filters: https://developer.mozilla.org/en-US/docs/Web/CSS/filter */
 #game-stream video {
-    filter: saturate(110%);
+    filter: saturate(110%) contrast(110%) brightness(110%);
```
Comment (redphx) [inline better-xcloud.user.js:24]: Could you please reverse this change and only keep the removal of warning dialog? Thank you.

**Comment by redphx** [comment]
@nathan130200 I've added the new Settings UI. Please update your code. Thank you.

**Pull request closed without merging**